src/lang/modifyAst/addFillet.ts

```diff
@@ -218,58 +207,20 @@ export function addFillet(
       return new Error('Expected an ObjectExpression node')
     }
 
-    if (filletTag === tag) {
```

Comment:
I ended up removing the edit case for the same reason as https://github.com/KittyCAD/modeling-app/pull/2718/commits/ad54158fc11f2d993f23c298449dfa6373d77343#r1678768111